### PR TITLE
Fixed extractions from dict vendors, [c.name] when is trying to parse…

### PR DESCRIPTION
**Pull request by dlgauss**
ted.py:
  Fixed 'vendors' extraction from dict
  Fixed fields[c.name] when it is trying to parse that value in int. Sometimes receiving ex. "2010."

settings.py
  added the 'workers' variable in order to make it possible to limit the number of workers for Q_CLUSTER. FTP from TED can only enable 3 simultaneous connections.

### CODE REVIEW

File: app/parsers/ted.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from bs4 import BeautifulSoup, element
 from datetime import date, datetime, timedelta
 from django.conf import settings
-from ftplib import error_perm, FTP
+from ftplib import error_perm, FTP,error_perm
```
Comment (arielpontes) [inline app/parsers/ted.py:10]: Duplicate import.

File: app/parsers/ted.py [hunk 2]
```diff
@@ -131,21 +135,23 @@ def parse_notices(
...
-
+        
         for archive_path in self.archives:
             try:
-                os.remove(archive_path)
+                if archive_path:
+                    os.remove(archive_path)
             except OSError as e:
-                logging.warning(e)
+                # logging.warning(e)
```
Comment (arielpontes) [inline app/parsers/ted.py:144]: I suppose this shouldn't be commented?

File: app/parsers/ted.py [hunk 3]
```diff
@@ -131,21 +135,23 @@ def parse_notices(
...
                 pass
 
         for folder in folders:
             try:
-                os.rmdir(os.path.join(self.path, folder))
+                if folder:
+                    os.rmdir(os.path.join(self.path, folder))
             except OSError as e:
-                logging.warning(e)
+                # logging.warning(e)
```
Comment (arielpontes) [inline app/parsers/ted.py:152]: Or this.

File: app/parsers/ted.py (hunk 1 above)
Comment (dlgauss) [reply app/parsers/ted.py:10]: Yes. I will fix.

File: app/parsers/ted.py (hunk 2 above)
Comment (dlgauss) [reply app/parsers/ted.py:144]: Yes, you are right. I will fix

File: app/parsers/ted.py (hunk 3 above)
Comment (dlgauss) [reply app/parsers/ted.py:152]: this one also, i will fix.

File: app/parsers/ted.py [hunk 4]
```diff
@@ -156,6 +163,7 @@ def extract_data(archive_path, extract_path):
             # Return the name of the folder containing the extracted files
             return tf.getnames()[0].split("/")[0]
         except (EOFError, FileNotFoundError) as e:
+            logging.warning('maybe in line 165')
```
Comment (arielpontes) [inline app/parsers/ted.py:166]: Leftover debugging log?

Comment (dlgauss) [reply app/parsers/ted.py]: This line should be deleted. I will fix it.

**Review by arielpontes — APPROVED** [review]

**Pull request merged**